### PR TITLE
[FEATURE] Added some features in filter and table in EditTests

### PR DESCRIPTION
**Pull request by sadoffToxir**
Added some features in filter and table in EditTests

### CODE REVIEW

File: src/components/EditTests/EditTestsTable/EditTestsTable.jsx [hunk 1]
```diff
@@ -51,9 +67,11 @@ export const EditTestsTable = (props) => {
           <Table stickyHeader aria-label='sticky table'>
             <TableHead>
               <TableRow>
-                {rows.map((rowName) => {
+                {filteredRows.map((rowName) => {
                   return (
-                    <TableCell key={rowName} align='left'>{Array.isArray(rowName) ? <Trans>{rowName[0]}{rowName[1]}</Trans> : rowName }</TableCell>
+                    <TableCell key={rowName} align='left' style={{ fontWeight: 700 }}>{Array.isArray(rowName)
```
Comment (snaf-trip) [inline src/components/EditTests/EditTestsTable/EditTestsTable.jsx]: why are the styles written here?

**Comment by sadoffToxir** [comment]
added classname to rows in table

File: src/components/EditTests/EditTestsTable/EditTestsTable.jsx [hunk 2]
```diff
@@ -62,13 +80,13 @@ export const EditTestsTable = (props) => {
...
+                  {
+                    row.module === 'Listening'
+                      ? <TableCell component='th' scope='row' size='small'>
+                        <PlayCircleOutlineIcon className='icons-color-primary' cursor='pointer' />
+                      </TableCell>
+                      : null
+                  }
                   <TableCell align='left' size='small'>{row.question}</TableCell>
                   <TableCell align='left'>
                     <Button color='primary' variant='outlined' size='small' style={{ width: 140 }} type='search' className='btn-search'>
```
Comment (snaf-trip) [inline src/components/EditTests/EditTestsTable/EditTestsTable.jsx:92]: Sorry, I didn't notice right away. Styles too.

Comment (snaf-trip) [reply src/components/EditTests/EditTestsTable/EditTestsTable.jsx:92]: I was talking about styles in the button))

File: src/pages/AdminDistribution/AdminDistribution.scss [hunk 3]
```diff
@@ -34,8 +34,8 @@
     min-width: 127px;
     max-height: 25px;
     font-weight: bold;
-    color: #3E6DD7;
-    border-color: #3E6DD7;
+    color: #3e6dd7;
+    border-color: #3e6dd7;
```
Comment (fshovchko) [inline src/pages/AdminDistribution/AdminDistribution.scss:38]: use theme selectors from styles/theme.scss

**Review by fshovchko — DISMISSED** [review]

File: src/components/EditTests/EditTestsFilter/EditTestsFilter.jsx [hunk 4]
```diff
@@ -72,6 +74,7 @@ export const EditTestsFilter = (props) => {
 };
 
 EditTestsFilter.propTypes = {
+  questId: PropTypes.any,
   setLevel: PropTypes.func,
```
Comment (mtalstykh-exadel) [inline src/components/EditTests/EditTestsFilter/EditTestsFilter.jsx:78]: questionID. No need to make the name shorter

**Review by fshovchko — APPROVED** [review]

**Review by mtalstykh-exadel — APPROVED** [review]

**Pull request merged**